dashboard/test/models/pd/enrollment_test.rb

```diff
@@ -134,4 +136,36 @@ class Pd::EnrollmentTest < ActiveSupport::TestCase
     enrollment.send_exit_survey
     assert_not_nil enrollment.reload.survey_sent_at
   end
+
+  test 'name is deprecated and calls through to full_name' do
+    enrollment = create :pd_enrollment
+    enrollment.expects(:full_name)
+    assert_deprecated 'name is deprecated. Use first_name & last_name instead.' do
+      enrollment.name
+    end
+
+    enrollment.expects('full_name=' => 'First Last')
+    assert_deprecated 'name is deprecated. Use first_name & last_name instead.' do
+      enrollment.name = 'First Last'
+    end
```

Comment:
Nice test syntax. Is `enrollment.expects` pure Mocha or is ActiveSomething also involved?